spec/support/generators.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module RestApiGenerator
+  module Specs
+    module Generators
+      module Macros
+        # Tell the generator where to put its output (what it thinks of as
+        # Rails.root)
+        def set_default_destination
+          destination DEFAULT_DESTINATION_PATH
+        end
+
+        def setup_default_destination
+          set_default_destination
+          before do
+            prepare_destination
+            # Create routes file
+            FileUtils.mkdir_p("#{DEFAULT_DESTINATION_PATH}/config")
+            File.open("#{DEFAULT_DESTINATION_PATH}/config/routes.rb", "w") do |f|
+              f.puts("Rails.application.routes.draw do")
+              f.puts("")
```

Comment:
```suggestion
```
The puts method adds a line break, this line is probably not needed

Reply:
I write this line to make the file more beautiful with the space like this, but yay, we dont need to keep it beatiful because its a tmp file:
```ruby
Rails.application.routes.draw do

end
```